en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,12 +1,18 @@
 = MuleSoft Composer Release Notes
-:page-revdate: April 12, 2023
+:page-revdate: April 17, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues for MuleSoft Composer] for a list of known issues by date.
 
+== April 17, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* The Error Handler (Beta) is now available. You can use this feature to watch for errors in connection or logic in specific steps of your flow and then designate logic to handle the errors that may occur.
```

Comment:
small comment: I would use other verb than "handle" to give some sort of example of what it means to "handle an error".

Maybe something like:
```The Error Handler (Beta) is now available. You can use this feature to watch for errors in connection or logic in specific steps of your flow and then designate logic to recover if an error occurs.```